backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -316,14 +319,19 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
 
     def evaluateEnvironmentExpression(nameAndExpression: (String, WomExpression)): ErrorOr[(String, String)] = {
       val (name, expression) = nameAndExpression
-      expression.evaluateValue(adHocFileCreationInputs, backendEngineFunctions) map { name -> _.valueString }
+      expression.evaluateValue(commandInstantiationInputs, backendEngineFunctions) map { name -> _.valueString }
     }
 
-    val environmentVariables = jobDescriptor.taskCall.callable.environmentExpressions.toList traverse evaluateEnvironmentExpression
+    val environmentVariables = callable.environmentExpressions.toList traverse evaluateEnvironmentExpression
+
+    import cats.instances.option._
```

Comment:
ToL I don't feel strongly but I personally like these to live at the top of the file, despite the broader scope.